torch_harmonics/distributed/distributed_resample.py

```diff
@@ -82,48 +81,51 @@ def __init__(
 
         # for upscaling the latitudes we will use interpolation
         self.lats_in, _ = _precompute_latitudes(nlat_in, grid=grid_in)
-        self.lons_in = np.linspace(0, 2 * math.pi, nlon_in, endpoint=False)
+        self.lons_in = torch.linspace(0, 2 * math.pi, nlon_in+1)[:-1]
```

Comment:
maybe  it would be good to pack this into a `_precompute_longitudes` routine? I see this pattern repeat manytimes and this should improve readility

Reply:
done